src/plugins/dashboard/public/application/dashboard_app_no_data.tsx

```diff
@@ -33,7 +33,7 @@ export const DashboardAppNoDataPage = ({
   };
   return (
     <AnalyticsNoDataPageKibanaProvider {...analyticsServices}>
-      <AnalyticsNoDataPage onDataViewCreated={onDataViewCreated} />;
+      <AnalyticsNoDataPage onDataViewCreated={onDataViewCreated} />
```

Comment:
Nice catch

Reply:
All credit to Caroline for finding those!

Re: snapshots - the dashboard snapshots did change in this PR because we tweaked panel drop shadow strength within EUI slightly. Unfortunately when I ran `--updateBaselines` on the FTR tests, my screenshots came out super large (2000px wide??) compared to the previous/CI screenshots. I'm not sure how to update the screenshots in a way that CI expects, really sorry 🙈 definitely happy to re-run the snapshot updates if there's any guidance you can give me in that direction!

Reply:
This isn't the first time I've heard of someone having trouble updating the baselines for these tests! Let me look into it later, but for the time being upping the threshold is okay! Thanks for letting me know - I might need to work with the operations team to get this resolved.